python/tk_multi_loader/model_publishhistory.py

```diff
@@ -82,6 +82,7 @@ def load_data(self, sg_data):
         filters = [ ["project", "is", sg_data["project"] ],
                     ["name", "is", sg_data["name"] ],
                     ["entity", "is", sg_data["entity"] ],
+                    ["sg_status_list", "not_in", ["omt", "ip"]],
```

Comment:
This shouldn't be necessary : our common tk-multi-loader2 settings are doing just that through "publish_filters"...